KinSDK/KinSDK/KinAccount.swift

```diff
@@ -365,20 +255,158 @@ public final class KinAccount {
     }
 }
 
+// MARK: Batch Payments
+
+extension KinAccount {
+    public func paymentQueue() -> PaymentQueue {
+        return nil!
+    }
+
+    public func pendingBalance(_ completion: @escaping (Result<Kin, Error>) -> Void) {
+        completion(.success(Kin(0)))
+    }
+
+    public func pendingBalance() -> Promise<Kin> {
+        return Promise(0)
+    }
+
+//    public func pendingBalanceListener() 
+}
+
 // MARK: - Deprecated
 
 extension KinAccount {
-    @available(*, deprecated, renamed: "buildPaymentTransaction")
-    public func generateTransaction(to recipient: String,
-                                    kin: Kin,
-                                    memo: String? = nil,
-                                    fee: Quark = 0,
-                                    completion: @escaping GenerateTransactionCompletion) {
-        return buildPaymentTransaction(to: recipient, kin: kin, memo: memo, fee: fee, completion: completion)
+    /**
+     Build a Kin transaction for a specific address.
+
+     The completion block is called after the transaction is posted on the network, which is prior
+     to confirmation.
+
+     - Attention: The completion block **is not dispatched on the main thread**.
+
+     - Parameter recipient: The recipient's public address.
+     - Parameter kin: The amount of Kin to be sent.
+     - Parameter memo: An optional string, up-to 28 bytes in length, included on the transaction record.
+     - Parameter fee: The fee in `Quark`s used if the transaction is not whitelisted.
+     - Parameter completion: A completion with the `PaymentTransaction` or an `Error`.
+     */
+//    @available(*, deprecated, renamed: "")
+    public func generateTransaction(to recipient: String, kin: Kin, memo: String? = nil, fee: Quark = 0, completion: @escaping GenerateTransactionCompletion) {
```

Comment:
generate transaction is the old name before linking account initiative? (trying to understand the renaming)

Reply:
yes, this is the name the devs are currently using